mappings/net/minecraft/inventory/InventoryClearable.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS afy net/minecraft/inventory/InventoryClearable
```

Comment:
As asie [commented on my PR](https://github.com/FabricMC/yarn/pull/460#discussion_r257184093), this should be `Clearable` since it's not restricted to inventories. But I mapped the class already so you can just remove it.

Reply:
Removed.